isort/main.py

```diff
@@ -23,6 +23,7 @@
 import os
 import re
 import sys
+from typing import Any, List  # noqa: F401
```

Comment:
`# noqa: F401` is here because of inability of `flake8==3.5.0` to parse type comments.